WordPress/Classes/Models/SiteType.swift

```diff
@@ -0,0 +1,31 @@
+
+/// A site type. There is already a SiteType enum in the codebase. To be renamed after we get rid of the old code
+struct SiteType {
+    let id: Identifier
```

Comment:
Given my Objective-C heritage, I'm always reluctant to name things `id`. What do you think about just calling it `identifier`?